addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -566,6 +610,60 @@
     </div>
 </template>
 
+<template id="modal_slot_registration" name="Modal for slots registration">
+    <div class="modal fade" id="modal_slot_registration" data-bs-backdrop="static" data-bs-keyboard="false"
+        tabindex="-1" aria-labelledby="staticBackdropLabel" aria-hidden="true">
+        <div class="modal-dialog modal-lg o_wevent_js_slot_details">
+            <form id="slot_registration_form" t-attf-action="/event/#{slug(event)}/registration/tickets" method="post">
+                <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
+                <div class="modal-content">
+                <div class="modal-header">
+                    <div class="o_wevent_registration_title modal-title fs-5" id="staticBackdropLabel">Slots</div>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body">
+                    <div class="d-flex flex-column flex-sm-row gap-sm-3">
+                        <h5 class="o_wevent_selected_slot_title">Select a Date:</h5>
+                        <span class="o_wevent_selected_slot text-primary fw-bold"/>
+                    </div>
+                    <nav class="mt-2">
+                        <ul class="pagination d-flex flex-nowrap gap-3 user-select-none">
+                            <li class="o_wevent_slot_previous page-item">
+                                <button class="btn btn-link text-decoration-none p-1">
+                                    <i class="fa fa-caret-left me-2"/><small>Previous</small>
+                                </button>
+                            </li>
+                            <li class="o_wevent_slot_next page-item">
+                                <button class="btn btn-link text-decoration-none p-1">
+                                    <small>Next</small><i class="fa fa-caret-right ms-2"/>
+                                </button>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div class="o_wevent_slot_dates row px-2">
+                        <div t-foreach="slots" t-as="date" t-attf-class="o_wevent_slot_date col col-sm-6 col-lg-4 px-3 pb-4">
+                            <time t-out="date" class="text-nowrap pe-5 me-5" t-options="{'widget': 'date', 'format': 'MMM dd YYYY, EEEE'}"/>
```

Comment:
Is it wise to hardcode date formats?

For example, for us, I think it will look like: "Mai 18 2025, Mercredi", which is not the usual way we read dates.